src/main/java/cz/codecamp/logger/fileHandlers/FileHandler.java

```diff
@@ -0,0 +1,113 @@
+package cz.codecamp.logger.fileHandlers;
+
+import java.io.File;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.TemporalUnit;
+
+/**
+ * Created by Filip Ocelka on 13.10.2016, filip.ocelka@gmail.com
+ */
+public abstract class FileHandler {
+
+    protected String fileName;
+    protected String logsFolder = "./logs";
+    protected LocalDateTime lastLogFileDateTime;
+    protected DateTimeFormatter formatter;
+
+    public FileHandler(String fileName, DateTimeFormatter formatter) {
+        this.fileName = fileName;
+        this.formatter = formatter;
+        lastLogFileDateTime = LocalDateTime.MIN;
+    }
+
+    public void setLogsFolder(String logsFolder) {
```

Comment:
Spíš než nastavovat složku přes setter bych to viděl na to mít ji v konstruktoru.